packages/v4/src/content/accessibility/page/page.md

```diff
@@ -0,0 +1,33 @@
+---
+id: Page
+section: components
+---
+
+The **page** component is used to define the basic layout of a page with either vertical or horizontal navigation. Page layouts are defined using page sections, such as the header, body, and footer of a page. The basic layout of a page depends on whether your page uses vertical or horizontal navigation.
+
+A page can be combined with many other components, so to create an accessible page, refer to the accessibility recommendations for each component.
+
+**Keyboard users** should be able to navigate through the masthead of the page and into other interactive elements of the page using **Tab** and **Shift + Tab** to move forward and backward through interactive elements. 
+
+**Screen reader users** should be able to navigate through and interact with the masthead of the page using the same interactions as keyboard users. Elements of a page that do not have visible descriptive text, such as toggle icons, should have alternative text for screen readers.
+
+Consider using a **skip to content** component on your page so that assistive technology users don’t need to make their way through the entire navigation menu each time they go to another page.
+
+## To make a page accessible:
+- Use roles to identify the main region and the banner region of a page.
+- Use aria-expanded on to indicate the state of the visibility of expandable content.
+- Use a tabindex of -1, as an entire page component should not be focusable by tabbing (though individual components within it may be).
```

Comment:
Since we add this for them, I don't think we should include it here. I think it will cause more confusion than anything.